sw/device/tests/penetrationtests/firmware/ibex_fi.c

```diff
@@ -586,6 +586,183 @@ status_t handle_ibex_fi_char_sram_read(ujson_t *uj) {
   return OK_STATUS();
 }
 
+status_t handle_ibex_fi_char_sram_write_static(ujson_t *uj) {
+  // Clear registered alerts in alert handler.
+  sca_registered_alerts_t reg_alerts = sca_get_triggered_alerts();
+
+  // Get address of buffer located in SRAM.
+  uintptr_t sram_main_buffer_addr = (uintptr_t)&sram_main_buffer;
+  mmio_region_t sram_region_main_addr =
+      mmio_region_from_addr(sram_main_buffer_addr);
+
+  // Initialize SRAM region with inverse ref_values to avoid that data from a
+  // previous run are still in memory.
+  for (int i = 0; i < 64; i++) {
+    mmio_region_write32(sram_region_main_addr, i * (ptrdiff_t)sizeof(uint32_t),
+                        ~ref_values[0]);
+  }
```

Comment:
Is there a problem with:
```suggestion
  // Initialize SRAM region with inverse ref_values to avoid that data from a
  // previous run are still in memory.
  for (size_t i = 0; i < 64; i++) {
    sram_main_buffer[i] = ~ref_values[0];
  }
```
?

Reply:
Yes, you are right, this works too and is much simpler. I've copied this from an SRAM test to make sure that the buffer is in SRAM.